### ISSUE TITLE
DOIs for publications

### ISSUE THREAD

**Issue by arfon** [openjournals/theoj#17]
Need to get DOIs for papers.

**Comment by arfon** [comment]
Let's use Zenodo for this.

**Comment by chrislintott** [comment]
Do we need, @arfon, to have references in a standard format to be allowed to issue DOIs, or do Zenodo cover that?

**Comment by arfon** [comment]
@chrislintott - we should talk more about this. I'm not sure that Zenodo is correct for this in the end as I don't think the papers will be indexed as ... papers.

/ cc @lnielsen

**Comment by lnielsen** [comment]
@arfon - True, you'll get DataCite DOIs from Zenodo instead of CrossRef DOIs - meaning different metadata schema, services, pricing etc. See e.g. http://www.crossref.org/02publishers/20pub_fees.html
You could possible get CrossRef DOIs through e.g. EZID from CDL (https://wiki.ucop.edu/display/DataCite/CrossRef+Service+FAQs + http://ezid.cdlib.org/home/pricing)

**Comment by marcrohloff** [comment]
We need to decide how to handle this.

**Comment by freelanceastro** [comment]
Agreed. @arfon, @chrislintott, any further thoughts on how we want to do this?
